contracts/libraries/VaultLifecycleSTETH.sol

```diff
@@ -569,4 +558,61 @@ library VaultLifecycleSTETH {
         (bool success, ) = payable(recipient).call{value: amount}("");
         require(success, "!success");
     }
+
+    function getOTokenPremium(
+        address oTokenAddress,
+        address optionsPremiumPricer,
+        uint256 premiumDiscount,
+        address collateralToken
+    ) external view returns (uint256) {
+        return
+            _getOTokenPremium(
+                oTokenAddress,
+                optionsPremiumPricer,
+                premiumDiscount,
+                collateralToken
+            );
+    }
+
+    function _getOTokenPremium(
```

Comment:
We had to duplicate this function across all the VaultLifecycle's because we have to link the 2 libraries otherwise. On top of that, we also do the extra calculation of multiplying the pricePerShare of the collateral token